packages/authenticator/amplify_authenticator/lib/src/models/authenticator_exception.dart

```diff
@@ -37,6 +38,12 @@ class AuthenticatorException extends AmplifyException {
     );
   }
 
+  const AuthenticatorException.customAuth()
```

Comment:
Out of scope for this PR - I think this could be removed as it appears to be unused.